lib/theme_check/language_server/completion_provider.rb

```diff
@@ -38,6 +38,16 @@ def doc_hash(content)
           },
         }
       end
+
+      def deprecated_hash(entry)
```

Comment:
When I read `deprecated_hash`, it sounds as if the hash is _always_ deprecated / as if the method would _always_ mark the hash as deprecated.
But this method feels more like `format_hash`, `format_hash_to_language_server_protocol`, `transform_hash`, …
WDYT?

Reply:
Excellent catch!! 🔥🔥🚀

Now, we're no longer missing `discount`:

![image](https://user-images.githubusercontent.com/1079279/207901638-c1e22905-2977-4e8f-bce6-bbafd9f78b30.png)